src/components/Alert/index.tsx

```diff
@@ -41,8 +56,31 @@ const Alert = ({
       className={cs(styles.base, styles[variant], className, classNames?.root)}
       {...props}
     >
-      {Icon && <Icon className={cs(styles.icon, classNames?.icon)} />}
-      <span className={cs(classNames?.message)}>{message}</span>
+      <div className={styles.container}>
+        <div className={styles.content}>
+          <Icon className={cs(styles.icon, classNames?.icon)} />
+          <p className={cs(styles.message, 'eyebrow', classNames?.message)}>
+            {message}
+          </p>
```

Comment:
if message is undefined, I don't think we should render this `p` element

Reply:
Before I change, the message was not with the? (optional) for the type. I ended up changing it to ? but I will leave it as it was.